dcos_installer/upgrade.py

```diff
@@ -12,14 +12,14 @@
 from pkgpanda.util import write_string
 
 
-node_upgrade_template = """
-#!/bin/bash
+node_upgrade_template = """#!/bin/bash
```

Comment:
This is required / better. SHEBANG line should be the first one.
In the current state.

```
$ head dcos_node_upgrade.sh

#!/bin/bash
#
# BASH script to upgrade DC/OS on a node
#
```